### PR TITLE
Adds (optional) inline certificate support for .ovpn files

### PR DESCRIPTION
**Pull request by conorsch**
Intended to supersede #55 by @urosorozel. 

Preserves existing role behavior, but for those who wish to simplify distribution of client configuration files, setting `openvpn_unified_client_profiles: yes` will cause the role to embed the certificate and CA information inline in the client `.ovpn` files.

@urosorozel, please review. @robbintt you may have thoughts, as well.

### CODE REVIEW

File: templates/client.conf.j2 [hunk 1]
```diff
@@ -38,9 +38,21 @@ persist-tun
...
+<ca>
+{{ openvpn_ca_file_contents }}
+</ca>
+<cert>
+{{ openvpn_read_client_cert_files_results.results|selectattr('item', 'equalto', client)|map(attribute='stdout')|list|first }}"
+</cert>
+<key>
+{{ openvpn_read_client_key_files_results.results|selectattr('item', 'equalto', client)|map(attribute='stdout')|list|first }}"
+</key>
+{% else %}
```
Comment (conorsch) [inline templates/client.conf.j2]: Haven't tested this with the ta.key introduced in #33; maybe that should be added here as well?

**Comment by urosorozel** [comment]
Great work Conorsch!

**Comment by conorsch** [comment]
Rebased on top of latest develop, to incorporate changes from recent merges.

**Comment by conorsch** [comment]
Thank you for taking a look, @urosorozel, and thanks to @msheiny for cleanup via #61. These changes are opt-in, and the new var is advertised in the README, so I'm going to merge once the CI checks pass.

**Pull request merged**